### DISCUSSION TITLE
Codeblock numbering per note

### DISCUSSION THREAD

**Discussion by pubdigital** [AnubisNekhet/AnuPpuccin#170] [Ideas]
Is there a way to disable Codeblock Numbering per note? I installed the Style Settings plugin and I can disable Codeblock Numbering globally, but when I take notes in some programming language the numbering is very useful, but not in other cases. For example when I take notes of my terminal sessions I would like to be able to disable it. Thanks in advance.

**Comment by AnubisNekhet** [comment]
Disable the option in settings then add `cssclass: anp-codeblock-numbers` to your YAML frontmatter in your note.
